### ISSUE TITLE
Build Tools and Dependency Management

### ISSUE THREAD

**Issue by YvanCywan** [DreamWeave-MP/Dreamweave#71]

**Comment by YvanCywan** [comment]
![Image](https://github.com/DreamWeave-MP/Dreamweave/assets/13180679/62f6472e-198e-49d3-8e20-cdd9d71f0c1c)

**Comment by magicaldave** [comment]
Packages provided by deps repo:

- Bullet
- LuaJIT
- MyGUI
- OSG
- OSGoS
- OpenAL
- SDL2
- Boost
- ffmpeg
- lz4
- qt5.qs ?
- lz4
- zlib

**Comment by YvanCywan** [comment]
Right, this is set up properly now, we might need to add some additional packages that don't yet have conan build scripts.

Goal for tomorrow is to talk about how we'd plan integrating this stuff into our CI.
